azure-pipelines.bundlesize.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         displayName: build packages & create reports
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26826&commitId=9c01b7ef1b24accfdfd73039e4d29e01969ac5c7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1257 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Context Menu.chromium.png|16562| Changed|
|Card.Card Example Expandable.chromium.png|17227| Changed|
|Card.Card Example Selectable Grid.chromium.png|113007| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>